guides/common/modules/proc_using-IdM-credentials-to-log-in-to-the-Project-Web-UI-with-a-Chrome-browser.adoc

```diff
@@ -0,0 +1,45 @@
+[id="Using_IdM_credentials_to_log_in_to_the_{Project}_Web_UI-with-a-Chrome-browser_{context}"]
+= Using IdM credentials to log in to the {ProjectWebUI} with a Chrome browser
+
+This section describes how to use a Chrome browser to log in to your {ProjectWebUI} with your {FreeIPA} (IdM) login and password.
+
+.Prerequisites
+* Your system administrator has enrolled your {ProjectServer} into IdM and configured the server to use IdM for authentication.
+For more information, see {InstallingServerDocURL}Configuring_External_User_Groups_{project-context}#Using_FreeIPA_admin[Using {FreeIPA}] in _{InstallingServerDocTitle}_.
+* An IdM server is running and reachable.
+* The host on which you are using the Chrome browser to log in to the {ProjectWebUI} is an IdM client.
+* You have a valid IdM login and password.
+* {Team} recommends using the latest stable Chrome browser.
+
+.Procedure
+. Open a terminal window.
+. Enable the Chrome browser to use Kerberos authentication:
++
+[options="nowrap", subs="+quotes,verbatim,attributes"]
+----
+google-chrome –auth-server-whitelist="*._example.com_" --auth-negotiate-delegate-whitelist=”*._example.com_"
+----
+
++
+[NOTE]
+====
+Instead of whitelisting the whole domain, you can also whitelist a specific {ProjectServer}.
```

Comment:
Can we use "allowlisting" here? Maybe check what IBM style guide says about this and blocklisting.

Reply:
I cannot see 'whitelist' in the 2012 version but yes, it is there in the newest version.
I will change the word.